packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -693,24 +693,36 @@ abstract class BaseMethod implements PythonBase {
     const liftedProperties: spec.Property[] = [];
 
     const stack = [this.liftedProp];
-    let current = stack.shift();
-    while (current !== undefined) {
+    const knownIfaces = new Set<string>();
+    const knownProps = new Set<string>();
+    for (
+      let current = stack.shift();
+      current != null;
```

Comment:
Why change from `undefined` to `null`. Per [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/shift#description), this will return `undefined` when empty?

Maybe just check for truthiness instead?

Reply:
`x != null` is the same as `x !== nul && x !== undefined`, but more compact.

And I don't like truthiness checks, because this is ripe for bugs.